backend/tests/cache/test_cache.py

```diff
@@ -0,0 +1,32 @@
+import pytest
+from time import sleep
+from backend.flaskr.restaurant_cache import RestaurantCache
+
+
+def test_add_restaurant():
+    # Initialize cache and test params
+    user = "Ben"
+    ID = "1234"
+    cache = RestaurantCache()
+    # Ensure adding a restaurant inserts it into cache
+    assert not cache.is_cached(user, ID)
+    cache.add_restaurant(user, ID)
+    assert cache.is_cached(user, ID)
+
+def test_clean_cache():
+    # Initialize cache and test params
+    timeout = 2
+    user = "Ben"
+    ID = "1234"
+    cache = RestaurantCache(timeout)
+    # Ensure the restaurant is removed only after timeout
+    cache.add_restaurant(user, ID)
+    cache.clean(user)
```

Comment:
There is already a clean step in cache.is_cached, so this can be removed.

Reply:
Good catch. I wasn't looking at my `is_cached()` method when I wrote that test and must have missed it.